src/test/java/seedu/address/logic/parser/ViewCommandParserTest.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.ViewCommand;
+
+public class ViewCommandParserTest {
+
+    private ViewCommandParser parser = new ViewCommandParser();
+
+    @Test
+    public void parse_validArgs_returnsViewCommand() {
+        assertParseSuccess(parser, " 1", new ViewCommand(INDEX_FIRST_PERSON));
```

Comment:
I like that you added a space at the side of "1" to test whether it trims it too.